src/box/box.cc

```diff
@@ -3436,12 +3436,17 @@ bootstrap_master(const struct tt_uuid *replicaset_uuid)
  * @param[out] start_vclock  the vector time of the master
  *                           at the moment of replica bootstrap
  */
-static void
+static int
 bootstrap_from_master(struct replica *master)
```

Comment:
Also, while we are at it: this looks super strange, that the function might throw and might return an error at the same time.

Let's better split it in two: I'm bad with names, but something like
`bootstrap_from_master_begin()`, which never throws, returns 0 when everything is fine and returns -1 when there was an error, which may be retried.
and
`bootstrap_from_master_end()` which returns nothing and throws an exception. Because any of its errors are unrecoverable anyway.

Reply:
Done